test/Fail/Issue2248_COMPILED_TYPE.err

```diff
@@ -3,7 +3,8 @@ Haskell types can only be given for postulates.
 
 error: [CompilationError]
 Compilation error:
-«path»/MAlonzo/Code/Issue2248_COMPILED_TYPE.hs:«line»:«col»: error: [GHC-25897]
+
+«path»/MAlonzo/Code/Issue2248_COMPILED_TYPE.hs:«line»:«col»: error:
```

Comment:
Oops, this was build with an version of GHC that did not have error names yet.
Unfortunately our test suite is not GHC-version agnostic.